### ISSUE TITLE
Unexpected block statement surrounding arrow body  arrow-body-style

### ISSUE THREAD

**Issue by jmls** [eslint/eslint#5471]
I get this error when linting the following code

```js
const gulp = require('gulp');
const conf = require('./conf');
const $ = require('gulp-load-plugins')();

gulp.task('css', () => {
  return gulp.src(`${conf.path.src}/**/*.css`)

    .pipe($.concatCss(conf.build.css))
    .pipe(gulp.dest('./dist/css'))
    .pipe($.notify({
      message: 'css task complete',
    }));
});

```

if I add a couple of lines (the let foo = true; foo = !foo)

```js
const gulp = require('gulp');
const conf = require('./conf');
const $ = require('gulp-load-plugins')();

gulp.task('css', () => {
  let foo = true;
  foo = !foo;

  return gulp.src(`${conf.path.src}/**/*.css`)

    .pipe($.concatCss(conf.build.css))
    .pipe(gulp.dest('./dist/css'))
    .pipe($.notify({
      message: 'css task complete',
    }));
});
```

then it lints without error

**Comment by erikdesjardins** [comment]
The rule is working as intended. You can safely remove the braces from your first example:

```js
gulp.task('css', () =>
  gulp.src(`${conf.path.src}/**/*.css`)
    .pipe($.concatCss(conf.build.css))
    .pipe(gulp.dest('./dist/css'))
    .pipe($.notify({
      message: 'css task complete',
    }))
);
```

But that's not possible when there are multiple statements in the arrow's body, which is why ESLint doesn't report any errors in your second example.

If you want ESLint to require that all arrow function bodies have braces, you need to use the `"always"` option; see [the documentation](http://eslint.org/docs/rules/arrow-body-style).

**Comment by jmls** [comment]
ah, thanks - I missed that.
